package.json

```diff
@@ -16,20 +16,22 @@
     "canvas": "^2.11.2",
     "cloudinary": "^1.33.0",
     "connect-flash": "^0.1.1",
-    "connect-mongo": "^4.6.0",
+    "connect-mongo": "^5.1.0",
+    "crypto": "^1.0.1",
```

Comment:
To be honest it seems a bit overkill to use crypto module here just to generate a uuid but we can let this one slide (there are good reasons for using it of course).